tables/hbo-cantillated.utb

```diff
@@ -0,0 +1,109 @@
+# liblouis: Classical Hebrew, Fully cantillated, IHBC-McAllister
+#
+#-index-name: Hebrew, classical, cantillated
+#-display-name: Classical Hebrew braille with full cantillation
+#
+#+language: hbo
+#+language: oar
+#+language: arc
+#+language: jpa
+#+language: obm
+#+language: xdm
+#+language: phn
+#+type: literary
+#+contraction: yes
```

Comment:
The `contraction` field normally has the values `no`, `full` or `partial`. Which of these matches best?